deployments/launcher/src/main/etc/indy/logging/logback.xml

```diff
@@ -134,9 +97,16 @@
   <!--
   <appender name="KAFKA" class="com.github.danielwegener.logback.kafka.KafkaAppender">
     <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
-      <layout class="ch.qos.logback.contrib.json.classic.JsonLayout">
+      <layout class="org.commonjava.indy.CustomJsonLayout">
         <jsonFormatter class="ch.qos.logback.contrib.jackson.JacksonJsonFormatter"/>
         <appendLineSeparator>true</appendLineSeparator>
+
+        <environmentMappings><![CDATA[
+          OPENSHIFT_BUILD_NAMESPACE=NAMESPACE,
+          OPENSHIFT_BUILD_NAME=BUILD_NAME,
+          OPENSHIFT_BUILD_COMMIT=BUILD_COMMIT,
+          HOSTNAME=HOSTNAME
+        ]]></environmentMappings>
       </layout>
     </encoder>
 
```

Comment:
I didn't see any problem in config, one concern is the portion between topic and producerConfig, they are out of date, the latest one should be in pnc-indy-etc/, if we cover config from that when pushing to prod, everthing should be ok, I just forgot to update this in previous commit.

Reply:
I went back and looked at our deployment config, and the only differences seem to be topic name and SSL configuration. I guess I could put in dummy versions of those SSL params, but I don't think I want to change the topic name since that's an infrastructural detail and not really relevant to the general case.